rxandroidble/src/test/groovy/com/polidea/rxandroidble/internal/connection/RxBleConnectionTest.groovy

```diff
@@ -655,6 +657,29 @@ class RxBleConnectionTest extends GradleRoboSpecification {
         setupIndicationUuidClosure             | setupNotificationCharacteristicClosure
     }
 
+    def "should queue custom operation"() {
+        given:
+        def operationBuilder = createCustomOperationBuilder()
+
+        when:
+        objectUnderTest.queue(operationBuilder).subscribe(testSubscriber)
+
+        then:
+        testSubscriber.assertCompleted()
+        testSubscriber.assertValues(true, false)
+    }
+
+    public createCustomOperationBuilder() {
+        new RxBleRadioOperationCustom<Boolean>() {
+            @Override
+            Observable<Boolean> execute(BluetoothGatt bluetoothGatt,
+                               RxBleGattCallback rxBleGattCallback,
+                               Scheduler scheduler) throws Throwable {
+                Observable.just(true, false)
+            }
+        }
+    }
+
```

Comment:
We could have more tests:
"should release the radio if custom operation will throw out of RxBleCustomOperation.execute()"
"should release the radio if observable returned from RxBleCustomOperation.execute() will emit error"
"should release the radio when observable returned from RxBleCustomOperation.execute() will complete"
"should pass error if observable returned from RxBleCustomOperation.execute() will emit error"
"should pass items emitted by observable returned from RxBleCustomOperation.execute()"
Maybe there are even more scenarios that should be covered.